.remote-config.js

```diff
@@ -1,4 +1,4 @@
 export const remoteConfig = {
-  apiUrl: 'dev-url',
-  conferenceLayerUrl: 'frame-url',
+  apiUrl: 'http://localhost:4000',
+  conferenceLayerUrl: 'http://localhost:8080',
```

Comment:
@carlossantos74 here it doesn't have to be:
  apiUrl: 'dev-url',
  conferenceLayerUrl: 'frame-url',

Reply:
@afonsovinicius I'll fix it, thx